go.mod

```diff
@@ -35,6 +21,10 @@ require (
 	github.com/bazelbuild/buildtools v0.0.0-20200922170545-10384511ce98
 	github.com/blang/semver/v4 v4.0.0
 	github.com/bwmarrin/snowflake v0.0.0
+	// Upstream is unmaintained. This fork introduces two important changes:
```

Comment:
Can you check that running `make update-go-deps` does not erase these comments?

Reply:
Just tried locally and it didn't change any files. :+1: